simplq/src/store/asyncActions/getQueueInfoByName.js

```diff
@@ -14,22 +14,17 @@ const useGetQueueInfoByName = () => {
   const makeAuthedRequest = useMakeAuthedRequest();
   const history = useHistory();
 
-  const getQueueInfoByName = createAsyncThunk(
-    typePrefix,
-    async ({ queueName }, { rejectWithValue }) => {
-      const authedRequest = makeAuthedRequest(RequestFactory.getQueueInfoByName(queueName));
+  return createAsyncThunk(typePrefix, async ({ queueName }, { rejectWithValue }) => {
+    const authedRequest = makeAuthedRequest(RequestFactory.getQueueInfoByName(queueName));
 
-      try {
-        const response = await authedRequest;
-        return response;
-      } catch (error) {
-        history.push(`/pageNotFound/queueName=${queueName}`);
-        return rejectWithValue({ message: `Queue ${queueName} does not exist, try again...` });
-      }
+    try {
+      const response = await authedRequest;
+      return response;
+    } catch (error) {
+      history.push(`/pageNotFound/queueName=${queueName}`);
+      return rejectWithValue({ message: `Queue ${queueName} does not exist, try again...` });
     }
-  );
-
-  return getQueueInfoByName;
```

Comment:
let's revert back to 
```
const getQueueInfoByName = createAsyncThunk(...) { ... };
return getQueueInfoByName;
```

Sorry about this, I will disable this rule on sonarqube

Reply:
Reverted the file